src/components/Breadcrumbs/Breadcrumbs.vue

```diff
@@ -0,0 +1,362 @@
+<!--
+ - @copyright Copyright (c) 2020 Raimund Schlüßler <raimund.schluessler@mailbox.org>
+ -
+ - @author Raimund Schlüßler <raimund.schluessler@mailbox.org>
+ -
+ - @license GNU AGPL version 3 or any later version
+ -
+ - This program is free software: you can redistribute it and/or modify
+ - it under the terms of the GNU Affero General Public License as
+ - published by the Free Software Foundation, either version 3 of the
+ - License, or (at your option) any later version.
+ -
+ - This program is distributed in the hope that it will be useful,
+ - but WITHOUT ANY WARRANTY; without even the implied warranty of
+ - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ - GNU Affero General Public License for more details.
+ -
+ - You should have received a copy of the GNU Affero General Public License
+ - along with this program. If not, see <http://www.gnu.org/licenses/>.
+ -
+ -->
+
+<docs>
+
+### General description
+
+This component renders a breadcrumb bar. It adjusts to the available width
+by hiding breadcrumbs in a dropdown menu and emits an event when something
+is dropped on a creadcrumb.
+
+### Usage
+
+```vue
+<template>
+	<div>
+		<div class="container container__wide">
+			<Breadcrumbs :breadcrumbs="breadcrumbs" @dropped="dropped" />
+		</div>
+		<br />
+		<div class="container container__narrow">
+			<Breadcrumbs :breadcrumbs="breadcrumbs" @dropped="dropped" />
+		</div>
+		<br />
+		<div class="dragme" draggable="true">
+			<span>Drag me onto the breadcrumbs.</span>
+		</div>
+	</div>
+</template>
+
+<script>
+export default {
+	computed: {
+		breadcrumbs() {
+			const path = 'Folder 1/Folder 2/Folder 3/Folder 4'
+			const crumbs = (path === '') ? [] : path.split('/')
+			return [{ name: 'Home', path: '/' }].concat(crumbs.map((crumb, i) => {
+				return {
+					name: crumb,
+					path: '/' + crumbs.slice(0, i + 1).join('/'),
+				}
+			}))
+		},
+	},
+	methods: {
+		dropped(path) {
+			alert('Dropped something on ' + path)
+		},
+	}
+}
+</script>
+<style>
+.container {
+	display: inline-flex;
+}
+
+.container__wide {
+	width: 800px;
+}
+
+.container__narrow {
+	width: 250px;
+}
+
+.dragme {
+	display: block;
+	width: 100px;
+	height: 44px;
+	background-color: var(--color-background-dark);
+}
+</style>
+```
+</docs>
+
+<script>
+import debounce from 'debounce'
+import { Actions } from '@nextcloud/vue/dist/Components/Actions'
+import { ActionRouter } from '@nextcloud/vue/dist/Components/ActionRouter'
+
+export default {
+	name: 'Breadcrumbs',
+	components: {
+		Actions,
+		ActionRouter,
+	},
+	props: {
+		rootIcon: {
+			type: String,
+			required: false,
+			default: 'icon-home',
+		},
+	},
+	data: function() {
+		return {
+			hiddenIndices: [],
+			hiddenCrumbs: [],
+			actionsOpen: false,
```

Comment:
Is this really needed?
You can also use the Actions as $ref and trigger open/close from there instead of keeping a duplicate v-model prop here :)

Reply:
I will check.